### ISSUE TITLE
What's the pytorch version

### ISSUE THREAD

**Issue by yihao31** [sshkhr/cmGAN#5]
I trained the model with pytorch 0.4.0 in Python3.6.3. The loss_G and loss_identity show a very large number.
![image](https://user-images.githubusercontent.com/22350011/54100341-fb628900-43f8-11e9-8729-53382c1a32d6.png)

**Comment by prismformore** [comment]
same problem here with pytorch 1.0. Anybody figures it out?

**Comment by sshkhr** [comment]
I also used pytorch 0.4.0 
I will check this issue after the ICCV deadline and get back to you

**Comment by yuzi56** [comment]
I got the same problem. The loss has been rising and finally turned into NaN.
Could you help me address it?  Thanks a lot.
Epoch --------------- 1
Epoch 001/2000 [0001/5071] -- loss_G: 15.6972 | loss_D: 1.3689 | loss_triplet_rgb: 2.6829 | triplet_loss_ir: 1.2321 | loss_identity: 11.7822 | loss_total: -15.6287 
-Epoch 001/2000 [0002/5071] -- loss_G: 16.1317 | loss_D: 1.3754 | loss_triplet_rgb: 2.3103 | triplet_loss_ir: 1.9274 | loss_identity: 11.8940 | loss_total: -16.0629 
-Epoch 001/2000 [0003/5071] -- loss_G: 17.5205 | loss_D: 1.3803 | loss_triplet_rgb: 2.1013 | triplet_loss_ir: 3.4595 | loss_identity: 11.9597 | loss_total: -17.4515 
-Epoch 001/2000 [0004/5071] -- loss_G: 74.9674 | loss_D: 1.5213 | loss_triplet_rgb: 22.0515 | triplet_loss_ir: 38.1460 | loss_identity: 14.7699 | loss_total: -74.8913
Epoch 001/2000 [0005/5071] -- loss_G: nan | loss_D: 4976841028734326669312.0000 | loss_triplet_rgb: nan | triplet_loss_ir: nan | loss_identity: 697867652548091466219
Epoch 001/2000 [0006/5071] -- loss_G: nan | loss_D: nan | loss_triplet_rgb: nan | triplet_loss_ir: nan | loss_identity: nan | loss_total: nan

**Comment by MykolaHavrylov** [comment]
@sshkhr Yeah, I have the same problem

![image](https://user-images.githubusercontent.com/56158562/140781035-1edce997-f7ff-4bfc-95c3-9e167a49e2e8.png)
